src/tokens.css

```
@@ -19,21 +21,141 @@
+  /* Border/surface color primitives (subtle alpha whites) */
+  --token-border-subtle: rgb(255 255 255 / 0.06);
+  --token-border-subtler: rgb(255 255 255 / 0.04);
+  --token-border-faint: rgb(255 255 255 / 0.08);
+  --token-surface-dim: rgb(255 255 255 / 0.04);
+
+  /* Chip color primitives */
+  --token-chip-active-bg: rgb(77 228 255 / 0.08);
+  --token-chip-tonic-bg: rgb(255 154 77 / 0.1);
+
   font-synthesis: none;
   text-rendering: optimizeLegibility;
```

Stylelint (17.7.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)